### PR TITLE
Added test for ip addr

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

File: test_ipaddr.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import re
+import unittest
+from testutils import system
```
Comment (kushaldas) [inline test_ipaddr.py]: Do a relative import.

```
from .testutils import system
```

File: test_ipaddr.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+class IpAddrTestCase(unittest.TestCase):
+
+    def test_system(self):
+        """
```
Comment (kushaldas) [inline test_ipaddr.py:8]: Change the function name to something more useful.

**Comment by trishnaguha** [comment]
@sudhirverma Change the filename to `testipaddr`. And classname to `IpAddrTest`.

File: test_ipaddr.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import unittest
+from .testutils import system
+
+class IpAddrTestCase(unittest.TestCase):
+
+    def test_ipaddr(self):
+        """
+        Test ip addr
+        """
+        out,err,ret = system("ip addr | grep 'inet '")
```
Comment (trishnaguha) [inline test_ipaddr.py:11]: You need to decode `out` using `utf-8` after the system command.

**Comment by trishnaguha** [comment]
@sudhirverma Any update on this?

**Comment by sudhirverma** [comment]
Added test for ip addr

**Pull request closed without merging**